src/Security/Authentication/Core/src/RemoteAuthenticationHandler.cs

```diff
@@ -48,12 +53,17 @@ public virtual async Task<bool> HandleRequestAsync()
                 return false;
             }
 
+            var authResult = await HandleRemoteAuthenticateAsync();
```

Comment:
This was pulled out of a `try`/`catch` block. Does that mean exceptions now behave differently?

Reply:
Oh, that's my bad! I didn't mean to change any behavior. I'll fix it.